pontoon/base/static/js/main.js

```diff
@@ -391,10 +391,9 @@ var Pontoon = (function (my) {
          * Get suggestions from machine translation and translation memory
          *
          * original Original string
-         * customSearch Instead of source string, use custom search keyword as input
          * loader Loader element id
          */
-        getMachinery: function (original, customSearch, loader) {
+        getMachinery: function (original, loader) {
```

Comment:
You'll also need to fix https://github.com/mozilla/pontoon/blob/a9aacb6ce812a9f95fbdc107986a416cfe193f96/pontoon/machinery/static/js/machinery.js#L11 for this. I haven't found more call sites?

Reply:
Nice catch. There is indeed only this call site remaining.

Which means we should move `getMachinery` to machinery.js. I could do also do it as part of this PR.

Reply:
I agree, and I think we should do this as part of this PR. Much of the changes are really about `getMachinery` not being a multi-purpose API anymore. Guess this is fallout from TN? That this used to be shared code between translate and machinery UI?

Reply:
That's correct. OK, I'll push a separate commit.

I've also identified several other unused pieces of code in main.js. These will go into a separate PR.

Reply:
I'll rebase #1851 once this is merged.

Reply:
No I won't.